src/views/Home/components/Schedule/schedule.scss

```diff
@@ -28,7 +28,7 @@
   &__schedule {
     position: relative;
     width: 75%;
-    min-width: min(500px, 70vw);
+    min-width: max(314.25px, 70vw);
```

Comment:
Let's double-check this parameter on very large screen sizes!

![image](https://user-images.githubusercontent.com/33771623/104823724-2f7f1000-5801-11eb-8ceb-2ef10a98ff46.png)